tests/integration/session_management/test_reservation.py

```diff
@@ -254,6 +254,94 @@ def test___sessions_reserved_by_nested_relay_group___get_connections_by_relays__
         ]
 
 
+def test___sessions_reserved___get_connections_by_pin_group___connections_returned(
```

Comment:
As per the Python [Regression Test Convention](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/20934/Regression-Test-Conventions?anchor=facts-and-theories), the `Then` clause should be something like `..._returns_connections`. 
I see that there are already inconsistencies b/w `connections_returned` and `returns_connections` but I think we should prefer and move forward with the right naming approach and it was also discussed sometime earlier.

Reply:
https://github.com/ni/measurementlink-python/pull/490#discussion_r1378807667